blog/en/blog/2023/12/26/access-kafka-by-apisix.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Access the Kafka Cluster by APISIX Gateway
+authors:
+  - name: Meng Yan
+    title: Author
+    url: https://github.com/yanmxa
+keywords:
+  - Message Queue
+  - Kafka
+  - Gateway
+  - Strimzi
+description: A few days ago, I tried to add a proxy to the kafka cluster, so I can leverage the proxy to manage the kafka authentication and authorization. Next, I will show how to use the APISIX to develop a customize authorization plugin for the kafka cluster.
+tags: [Ecosystem]
+---
+
+> This blog shows how to use Apache APISIX to develop a customize authorization plugin for the kafka cluster.
+
+<!--truncate-->
+
+## Prerequisites
+
+- Openshift cluster
+- Kafka cluster created by [strimzi kafka operator](https://github.com/strimzi/strimzi-kafka-operator)
+- kubectl, [oc](https://docs.openshift.com/container-platform/4.11/cli_reference/openshift_cli/getting-started-cli.html) and curl
+
+## Expose the Kafka Cluster by KafkaBridge
+
+To simplify the configuration setting for the kafka. I provision the kafka by [strimzi-kafka-operator](https://github.com/strimzi/strimzi-kafka-operator). In order to make Kafka expose interfaces externally like other services, I use `KafkaBridge` to transform it into an HTTP service.
+
+- Create the `KafkaBridge`
```

Comment:
A general suggestion: no need to use bullet points for each of the steps; you could write full sentences.